include/jemalloc/internal/background_thread_structs.h

```diff
@@ -19,6 +19,9 @@
 #define BACKGROUND_THREAD_HPA_INTERVAL_MAX_UNINITIALIZED (-2)
 #define BACKGROUND_THREAD_HPA_INTERVAL_MAX_DEFAULT_WHEN_ENABLED 5000
 
+#define DEFERRED_ASAP UINT64_C(0)
```

Comment:
Sure, thanks for pointing out!

Reply:
Maybe `DEFERRED_MIN`?

Reply:
I wanted these constants to answer the question `when should deferred work happen?`. But you're right, in this case they should contain a time rather than an interval.

If I rename it to `BACKGROUND_THREAD_DEFERRED_MIN`, then I should also rename `DEFERRED_NEVER` to `BACKGROUND_THREAD_DEFERRED_MAX`.